pool/pool_manager.gno

```diff
@@ -220,46 +216,91 @@ func DoesPoolPathExist(poolPath string) bool {
 	return exist
 }
 
-// GetPool retrieves the pool for the given token paths and fee.
-// It constructs the poolPath from the given parameters and returns the corresponding pool.
-// Returns pool struct
+// GetPool retrieves a pool instance based on the provided token paths and fee tier.
+//
+// This function determines the pool path by combining the paths of token0 and token1 along with the fee tier,
+// and then retrieves the corresponding pool instance using that path.
+//
+// Parameters:
+// - token0Path (string): The unique identifier or path for token0.
```

Comment:
`path` is unique `identifier`. 

```suggestion
// - token0Path (string): The unique path for token0.
```

Reply:
@r3v4s 
https://github.com/gnoswap-labs/gnoswap/pull/442/commits/bb7be92dc62d69ea2f5dd69327ec547198c6bac1